scripts/pages/zoning/zoning.ts

```diff
@@ -449,7 +456,7 @@ class ZoneMenuHandler {
 	populateDropdown(array: any[], dropdown: DropDown, optionType: string, clearDropdown: boolean = false) {
 		if (clearDropdown) dropdown.RemoveAllOptions();
 
-		for (const [i, item] of array.entries()) {
+		for (const [i, item] of array?.entries()) {
 			this.addOptionToDropdown(optionType || item, dropdown, i, optionType !== '');
 		}
```

Comment:
This I don't think is safe, however. If array is `null`, your elvis will propagate it along, and you'll get a JS exception trying to iterate over `null`.